Sources/Rendering/Core/Mapper/CoincidentTopologyHelper.js

```diff
@@ -29,7 +29,10 @@ const staticOffsetModel = {
   Line: { factor: 1, offset: -1 },
   Point: { factor: 0, offset: -2 },
 };
-const staticOffsetAPI = {};
+const noOp = () => undefined;
+const staticOffsetAPI = {
+  modified: noOp,
```

Comment:
why bother setting the key value to `modified` if it will be ignored  line 65 ?

Reply:
> When calling the static mapper vtkMapper.setResolveCoincidentTopologyPolygonOffsetParameters, there is an error because the "class" does not have a publicAPI with a modified() function.

Reply:
gotcha, you might want to add a "comment" in the code to make it less confusing.

Reply:
Added comment.